pinch/T1B3_GAP.DATA

```diff
@@ -0,0 +1,282 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2023 Equinor
+
+-- Test model for MINPV and PINCH. Grid with dimension 1x1x4 with cells in layer 
+-- 2 and 3 inactive due to minpv.   
```

Comment:
We should say "..in layer**s** 2 and 3..." here–i.e., plural instead of singular.